app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = list()
+    for person in people:
+        if None in person.values():
+            new_list.append(Person(person["name"], person["age"]))
+        else:
+            keys = list(person.keys())
```

Comment:
`person.keys()` == `person`

therefore it is not necessary to use `keys()` here